mlblocks_primitives/keras.Sequential.LSTMTimeSeriesRegressor.json

```diff
@@ -0,0 +1,114 @@
+{
+    "name": "keras.Layers.LSTM",
+    "author": "Ihssan Tinawi <itinawi@mit.edu>",
+    "documentation": "https://keras.io/layers/recurrent/#lstm",
+    "description": "Long Short Term Memory Layer",
+    "classifiers": {
+        "type": "estimator",
+        "subtype": "regressor"
+    },
+    "modalities": [],
+    "primitive": "mlprimitives.adapters.keras.Sequential",
+    "fit": {
+        "method": "fit",
+        "args": [
+            {
+                "name": "X",
+                "type": "ndarray"
+            },
+            {
+                "name": "y",
+                "type": "array"
+            }
+        ]
+    },
+    "produce": {
+        "method": "predict",
+        "args": [
+            {
+                "name": "X",
+                "type": "ndarray"
+            }
+        ],
+        "output": [
+            {
+                "name": "y",
+                "type": "array"
+            }
+        ]
+    },
+    "hyperparameters": {
+        "fixed": {
+            "dense_units": {
```

Comment:
This input is usually required for classification networks, so I'm not sure if this is needed here.
If it is, perhaps the primitive should be a Classifier rather than a Regressor?